apps/lockfile-explorer-web/tsconfig.json

```diff
@@ -1,3 +1,6 @@
 {
-  "extends": "./node_modules/local-web-rig/profiles/app/tsconfig-base.json"
+  "extends": "./node_modules/local-web-rig/profiles/app/tsconfig-base.json",
+  "compilerOptions": {
+    "skipLibCheck": true
```

Comment:
This is getting errors building in dependencies from some pnpm confusion around the new `@types/react` package